Improve error message on Features ContentCrs parameter checks

Our server incorrectly returns the following header:

``
Content-Crs: http://www.opengis.net/def/crs/OGC/1.3/CRS84
``

As you can see, angle brackets are missing. And yet, the error message returned by the test suite is:

``
java.lang.AssertionError: Features response at 'https://gs-main.geosolutionsgroup.com/geoserver/ogc/features/v1/collections/tiger:poly_landmarks/items?f=application%2Fgeo%2Bjson' does not provide expected 'Content-Crs' header, was: 'http://www.opengis.net/def/crs/OGC/1.3/CRS84', expected: 'http://www.opengis.net/def/crs/OGC/1.3/CRS84'
``

It should be saying that ``expected: '<http://www.opengis.net/def/crs/OGC/1.3/CRS84>'`` instead.

We spent a few minutes wondering why it was complaining that it received exactly what it was expecting, then checked the spec again and figured out the missing angle brackets.

--- a/src/main/java/org/opengis/cite/ogcapifeatures10/conformance/crs/query/crs/feature/FeatureCrsParameter.java
+++ b/src/main/java/org/opengis/cite/ogcapifeatures10/conformance/crs/query/crs/feature/FeatureCrsParameter.java
@@ -90,7 +90,7 @@ public class FeatureCrsParameter extends AbstractFeatureCrs {
                                                      featureUrl ) );
         }
         assertCrsHeader( actualHeader, crs,
-                         String.format( "Features response at '%s' does not provide expected 'Content-Crs' header, was: '%s', expected: '%s'",
+                         String.format( "Features response at '%s' does not provide expected 'Content-Crs' header, was: '%s', expected: '<%s>'",
                                         featureUrl, actualHeader, crs ) );
     }
 

--- a/src/main/java/org/opengis/cite/ogcapifeatures10/conformance/crs/query/crs/features/FeaturesCrsParameter.java
+++ b/src/main/java/org/opengis/cite/ogcapifeatures10/conformance/crs/query/crs/features/FeaturesCrsParameter.java
@@ -66,7 +66,7 @@ public class FeaturesCrsParameter extends AbstractFeaturesCrs {
                                                      featuresUrl ) );
         }
         assertCrsHeader( actualHeader, crs,
-                         String.format( "Features response at '%s' does not provide expected 'Content-Crs' header, was: '%s', expected: '%s'",
+                         String.format( "Features response at '%s' does not provide expected 'Content-Crs' header, was: '%s', expected: '<%s>'",
                                         featuresUrl, actualHeader, crs ) );
     }
 

